pkg/pmem-grpc/grpc.go

```diff
@@ -41,6 +42,11 @@ func Connect(endpoint string, tlsConfig *tls.Config) (*grpc.ClientConn, error) {
 	} else if proto == "unix" {
 		dialOptions = append(dialOptions, grpc.WithDialer(unixDialer))
 	}
+	// This is necessary when connecting via TCP and does not hurt
+	// when using Unix domain sockets. It ensures that gRPC detects a dead connection
+	// in a timely manner.
+	// code lift from https://github.com/kubernetes-csi/csi-test/commit/6b8830bf5959a1c51c6e98fe514b22818b51eeeb
+	dialOptions = append(dialOptions, grpc.WithKeepaliveParams(keepalive.ClientParameters{PermitWithoutStream: true}))
```

Comment:
s/code lift/Code lifted/

There's also a typo in your commit message (connwction).

Reply:
Done.